nncf/common/composite_compression.py

```diff
@@ -322,7 +322,10 @@ def export_model(self,
             assert backend is BackendType.TORCH
             from nncf.torch.exporter import PTExporter #pylint: disable=cyclic-import
             exporter = PTExporter(self.model, input_names, output_names, model_args)
-        exporter.export_model(save_path, save_format)
+        if save_format is not None:
```

Comment:
The `export_model` functions were poorly designed so that passing `None` as `save_format` triggered usage of the default parameter. This means, among other things, that we cannot gather this parameter explicitly. We probably don't want to see `None` in the telemetry readings as the values for the "save_format" event. I then went ahead and set the argument of the `save_format` to specify correct default value, uncovering the places in code where `None` was passed to the function on purpose to sidestep the wrong design by more wrong design. I corrected the code that used the function along with the function signatures.

Reply:
Thanks for the detailed answer. The changes make sense.